coordinator/src/main/java/org/apache/uniffle/coordinator/web/resource/ServerResource.java

```diff
@@ -27,7 +27,7 @@
 import java.util.stream.Stream;
 import javax.servlet.ServletContext;
 
-import org.apache.commons.collections4.CollectionUtils;
+import org.apache.commons.collections.CollectionUtils;
```

Comment:
This should be reverted. We use collections4 instead of collections.

Reply:
Done